tests/resources/conformance_suites/nl_nt18/fr_nl/4-02-unused-unit.xbrl

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xbrl
+  xml:lang="en"
+  xmlns="http://www.xbrl.org/2003/instance"
+  xmlns:iso4217="http://www.xbrl.org/2003/iso4217"
+  xmlns:jenv-bw2-i="http://www.nltaxonomie.nl/nt16/jenv/20211208/dictionary/jenv-bw2-data"
+  xmlns:link="http://www.xbrl.org/2003/linkbase"
+  xmlns:xbrldi="http://xbrl.org/2006/xbrldi"
+  xmlns:xbrli="http://www.xbrl.org/2003/instance"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+    <link:schemaRef xlink:href="http://www.nltaxonomie.nl/nt16/jenv/20211208/dictionary/jenv-bw2-data.xsd" xlink:type="simple"/>
```

Comment:
Since these testcases are importing nt16, which are in the default cache, they will pass in CI. You're OK to add them to the index.

Reply:
Aaron and Allen are working on this in separate PRs.